ctapipe_io_magic/__init__.py

```diff
@@ -80,6 +82,75 @@
     DATA_MAGIC_LST_TRIGGER: EventType.SUBARRAY,
 }
 
+class ReportLaserContainer(Container):
+    """ Container for Magic laser parameters """
+    UniqueID = Field(None, 'No.')
+    Bits = Field(None, 'ID')
+    MJD = Field(None, 'Modified Julian Date')
+    BadReport  = Field(bool, 'Bad Report')
+    State = Field(None, 'State')
+    IsOffsetCorrection = Field(bool, 'Is Offset Correction')
+    IsOffsetFitted = Field(bool, 'Is Offset Fitted')
+    IsBGCorrection = Field(bool, 'Is BG Correction')
+    IsT0ShiftFitted = Field(bool, 'Is T0 Shift Fitted')
+    IsUseGDAS = Field(bool, 'Is Use GDAS')
+    IsUpwardMoving = Field(bool, 'Is Upward Moving')
+    OverShoot = Field(nan, 'Over Shoot')
+    UnderShoot = Field(nan, 'Under Shoot')
+    BGSamples = Field(np.float32(np.nan), 'BG Samples')
```

Comment:
According to the comment below, these fields (plus other below) are also arrays in general.

Reply:
I changed the fields into arrays.